assets/src/edit-story/components/form/media.js

```diff
@@ -63,6 +65,16 @@ function MediaInput(
     type,
   });
 
+  const openCropper = useImageCrop({
+    title,
+    buttonInsertText,
+    onSelect: onChange,
+    onSelectErrorMessage: onChangeErrorText,
+    params,
+  });
+
+  const openDialog = params ? openCropper : openMediaPicker;
```

Comment:
Can't we just move this logic into `useMediaPicker`?

That said, not sure why `useImageCrop` and `useMediaPicker` are separate. Can't that code all live in `useMediaPicker`?

Reply:
The logic is different that they makes sense for them to be in different hooks.